Discovery/config_dialog.py

```diff
@@ -135,24 +214,71 @@ def populate_columns(self):
             for column in columns:
                 cbo.addItem(column)
 
+    def delete_config_from_settings(self, key, settings):
+        settings.remove(key + "connection")
+        settings.remove(key + "schema")
+        settings.remove(key + "table")
+        settings.remove(key + "search_column")
+        settings.remove(key + "echo_search_column")
+        settings.remove(key + "display_columns")
+        settings.remove(key + "geom_column")
+        settings.remove(key + "scale_expr")
+        settings.remove(key + "bbox_expr")
+
+        #settings.remove(key + "marker_time_enabled")
+        #settings.remove(key + "marker_time")
+
+
+    def validate_key(self, key, config_list):
+
+        if len(key) < 3: return False
```

Comment:
Is there a reason why it needs at least 3 characters? Accepting any non-empty string would be probably fine...